eclipse/tern.eclipse.ide.core/src/tern/eclipse/ide/core/TernCorePlugin.java

```diff
@@ -81,7 +81,8 @@ public static File getTernCoreBaseDir() throws IOException {
 
 	public static File getTernBaseDir() throws IOException {
 		return new File(FileLocator.getBundleFile(Platform
-				.getBundle(ternjs.Activator.PLUGIN_ID)), "node_modules/tern");
+				.getBundle(ternjs.Activator.PLUGIN_ID)), "node_modules/tern")
+				.getCanonicalFile();

```

Comment:
Why calling getCanonicalFile(). here? What is the problem you are fixing?

For your information getTernBaseDir was removed and replaced with getTernRepositoryBaseDir (root folder of tern repository).